### PR TITLE
Added client page

### PR DESCRIPTION
**Pull request by dosorno7**
I added the client page, also moved the PNG's all into one assets folder for ease of future access and organization. Made some slight changes to names as well to avoid any possible confusion.

### CODE REVIEW

**Review by daniel-henderson1 — APPROVED** [review]
LGTM

**Review by Pratheekl22 — APPROVED** [review]
LGTM

File: react-app/src/containers/HomePage/HomePage.css [hunk 1]
```diff
@@ -61,7 +61,7 @@ body {
     width: 35px;
     height: 35px;
 
-    background: url(icons8-male-user-96.png);
+    background: url(../Assets/Imagesicons8-male-user-96.png);
```
Comment (lizcooper44) [inline react-app/src/containers/HomePage/HomePage.css:64]: Need to have a / between Images and image name

**Review by lizcooper44 — CHANGES_REQUESTED** [review]
Just need to add a / to the new route for one of the images (I left a comment) otherwise lgtm

File: react-app/src/containers/HomePage/HomePage.css (hunk 1 above)
Comment (dosorno7) [reply react-app/src/containers/HomePage/HomePage.css:64]: Thanks for catching that, change made and pushing now

**Review by lizcooper44 — APPROVED** [review]

**Pull request merged**